doc/source/whatsnew/v0.21.0.txt

```diff
@@ -40,7 +40,8 @@ Other Enhancements
 - :func:`DataFrame.clip()` and :func:`Series.clip()` have gained an ``inplace`` argument. (:issue:`15388`)
 - :func:`crosstab` has gained a ``margins_name`` parameter to define the name of the row / column that will contain the totals when ``margins=True``. (:issue:`15972`)
 - :func:`Dataframe.select_dtypes` now accepts scalar values for include/exclude as well as list-like. (:issue:`16855`)
-
+- :func:`read_csv`, :func:`read_html`, :func:`read_json`, :func:`read_html`  now accept auth in url //<user>:<password>@<host>:<port>/<url-path>, or ``auth`` tuple (username, password) parameter 
+- :func:`read_csv`, :func:`read_html`, :func:`read_json`, :func:`read_html`  now accept ``verify_ssl`` False to disable https/ssl certificate verification (eg: self signed ssl certs in testing)  (:issue:`16716`)
```

Comment:
I'm personally still in favor of providing something more general like what I had suggested before and have a section explaining what you can do now.

@jreback : Thoughts?